### ISSUE TITLE
Doesn't work in TB 78

### ISSUE THREAD

**Issue by BillH99999** [eykamp/Folder-Account#11]
Hi Chris,

I notice that the plugin doesn't work in TB 78.  Are there plans to update it so it will work?  If so, any idea when that might be released.

Love the plugin and would hate to lose the functionality it provides.

Thanks,
Bill

**Comment by Welpy-cw** [comment]
Hi Bill,

I spent the last several days trying to adapt the add-on to Thunderbird 78. Using the Guide from John Bieling and his WindowListener API in the end not too much new code was necessary. The most noticable change is the new "Folder Account" tab in the folder properties containing the settings. 

However, thorough testing is recommended before using it in production!

https://github.com/Welpy-cw/Folder-Account/releases/tag/v10.0-pre

Please report any bugs to me.

**Comment by blacklion** [comment]
Works for me, but lost all settings from previous version. But is much better than nothing, thank you!

**Comment by msebald** [comment]
> Works for me, but lost all settings from previous version. But is much better than nothing, thank you!

It works for me, too. And all my settings (at least the folders I set up - did not check anything else, yet) are still there!

Thank you very much for the adaption to TB 78!

**Comment by lgbaldoni** [comment]
@Welpy-cw if the original author is unresponsive, how about submitting your version as a new add-on?

**Comment by Welpy-cw** [comment]
Looks like I'll have to do exactly that ...

**Comment by BillH99999** [comment]
@Welpy-cw 

I've been using version 10 since you posted the link and have not run into any problems.  It is working great for me.  I look forward to it being a new add-on.

Thanks for doing this... I don't know how I'd survive without this.

Bill

**Comment by Welpy-cw** [comment]
I just submitted my adaptation myself to ATN, it is currently awaiting review, yet already accessible directly from https://addons.thunderbird.net/en-US/thunderbird/addon/folder-account-2020/

**Comment by BillH99999** [comment]
@Welpy-cw 

Thanks!!

Bill

**Comment by eykamp** [comment]
Hi @Welpy-cw -- Thanks for your work on Folder Account.  Would you please contact me via email?  chris@eykamp.com.  Thanks!

**Comment by Welpy-cw** [comment]
As I'm taking over maintenance for this add-on for the time being, please refer to https://github.com/Welpy-cw/Folder-Account for all subsequent issues. Version 10.0 for Thunderbird 78, which is currently awaiting approval on ATN, is available there as well, see https://github.com/Welpy-cw/Folder-Account/releases/tag/v10.0

Thank you, Chris, for the development of this indispensable extension!

**Comment by BillH99999** [comment]
@Welpy-cw

Thanks again for doing this.

Will it now be called Folder Account instead of Folder Account 2020?  I don't see version 10 of either on ATN.

Also, I have another question.  In the description of the plugin at https://github.com/Welpy-cw/Folder-Account, it says "Please submit issues at the project's GitHub page".  I don't see an Issues tab or option as there is on most plugin pages.  Are you going to be adding one?  By the way, I don't have any issues at the moment.

Thanks,
Bill

**Comment by Welpy-cw** [comment]
@BillH99999 

Yes, it will be keeping its original name, therefore I retracted the forked "2020" listing. I uploaded v10.0 on the original https://addons.thunderbird.net/en-US/thunderbird/addon/folder-account page, it's awaiting review and hopefully will be published soon.

Thank you for your valuable hint, "Issues" were disabled, but should be available now.

**Comment by BillH99999** [comment]
@Welpy-cw 

Which is the better place to report issues, https://github.com/Welpy-cw/Folder-Account or https://github.com/eykamp/Folder-Account?

Thanks,
Bill

**Comment by Welpy-cw** [comment]
https://github.com/Welpy-cw/Folder-Account

**Comment by BillH99999** [comment]
Will do.

If I go into my add-ons Manager in TB and select Folder Account (currently version 10.0) it says:

NEW: The code for this project is now available at **https://github.com/eykamp/Folder-Account**. While I am no longer actively maintaining the project, I will accept pull requests and will update the extension as needed.

and it says:

PLEASE, do not use this page for support or feature requests. I am more than happy to help you, but I cannot respond to your comments here. Please submit issues at the project's GitHub page: **https://github.com/eykamp/Folder-Account**

Shouldn't these be changed then to say https://github/Welpy-cs/Folder-Account.

It is a bit confusing to have both of these sites.

Bill

**Comment by Welpy-cw** [comment]
@BillH99999 

I already updated that description on the ATN page of Folder Account. If you remove the extension in Thunderbird and reinstall the current version from https://github.com/Welpy-cw/Folder-Account/releases/tag/v10.0 that updated description should be shown.

Sorry for the confusion! Once the new version is approved, I hope it straightens out.

**Comment by BillH99999** [comment]
I downloaded and installed version 10 from the link you provided.  When I go to the Add-ons Manager it still shows the two references.

![image1](https://user-images.githubusercontent.com/26846819/96006956-dfa03f80-0df2-11eb-8790-8f2afe010c9d.jpg)

**Comment by Welpy-cw** [comment]
@BillH99999 Whenever an add-on is installed for the first time, Thunderbird retrieves information from its ATN page (under "About this Add-on") and shows it under "Details". This information is cached in TB's internal database and for some reason only updated by the (by default) daily background update check for installed add-ons. Manually checking for updates or reinstalling the add-on doesn't trigger this.

So if I'm not mistaken, your Thunderbird installation should show the updated info in a few hours. If not, please report, then I'll investigate further!

**Comment by BillH99999** [comment]
I checked just now and I am seeing the updated description now. 

Thanks!
Bill
